includes/functions.php

```diff
@@ -1113,11 +1113,12 @@ function port_fill_missing(&$port, $device)
         // ifAlias overridden by user, don't update it
         unset($port['ifAlias']);
         d_echo(' ifAlias overriden by user');
-    } elseif ($port['ifAlias'] == '' || $port['ifAlias'] == null) {
-        $port['ifAlias'] = $port['ifDescr'];
-        d_echo(' Using ifDescr as ifAlias');
+    } elseif ($device['os'] != 'fabos') {
```

Comment:
Why do you need to put an OS check here?

Reply:
I've done this to prevent ifAlias eventlog entries after every discovery run

Reply:
Ideally, you would fetch this ifAlias alternative there too.

Reply:
Added fill ifAlias for fibrechannel ports also to discovery process and removed the exception in functions.php